extensions/federated-catalog/store/fcc-node-directory-cosmos/build.gradle.kts

```diff
@@ -16,16 +16,15 @@ plugins {
     `java-library`
 }
 
-val cosmosSdkVersion: String by project
-val failsafeVersion: String by project
-
 dependencies {
     api(project(":core:common:util"))
     api(project(":spi:federated-catalog:federated-catalog-spi"))
     api(project(":extensions:common:azure:azure-cosmos-core"))
 
-    implementation("com.azure:azure-cosmos:${cosmosSdkVersion}")
-    implementation("dev.failsafe:failsafe:${failsafeVersion}")
+    implementation(
+        libs.azure.cosmos
+    )
```

Comment:
Just wondering: is there a reason why this is always a three-liner for `libs.azure.cosmos`? (didn't find that for any other dependency)

Reply:
good question, don't think so. it might have been autoformatted